bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.c

```diff
@@ -372,14 +369,7 @@ void bsg_serialize_device(const bsg_device_info device, JSON_Object *event_obj)
 }
 
 void bsg_serialize_device_metadata(const bsg_device_info device, JSON_Object *event_obj) {
-  json_object_dotset_string(event_obj, "metaData.device.brand", device.brand);
-  json_object_dotset_boolean(event_obj, "metaData.device.emulator", device.emulator);
   json_object_dotset_boolean(event_obj, "metaData.device.jailbroken", device.jailbroken);
```

Comment:
I believe this should be in `device.jailbroken`, not metadata.

Reply:
I've updated the serializer to put this in `device.jailbroken` instead.